index.html

```diff
@@ -1323,23 +1336,22 @@ <h3 id="name-request-parameters">
+<p id="section-2.3-4">The key used to sign the request object MUST validate to that featured in the <code>sub_jwk</code> parameter of the request.<a href="#section-2.3-4" class="pilcrow">¶</a></p>
+<p id="section-2.3-5">Unsigned plaintext Request Objects, containing <code>none</code> in the <code>alg</code> value of the JOSE header MUST not be supported.<a href="#section-2.3-5" class="pilcrow">¶</a></p>
+<p id="section-2.3-6">If the Request Object signing validation fails or is missing, the OpenID Connect Provider MUST respond to the request with the Error Response parameter, <a href="https://openid.net/specs/openid-connect-core-1_0.html#AuthError">section 3.1.2.6.</a> with Error code: <code>invalid_request_object</code>.<a href="#section-2.3-6" class="pilcrow">¶</a></p>
+<p id="section-2.3-7">If the <code>did</code> value is present in the request and the OpenID Provider does not support the usage of <a href="https://w3c.github.io/did-core/">decentralized identifiers</a> the value should be ignored.<a href="#section-2.3-7" class="pilcrow">¶</a></p>
 </section>
 </div>
 <div id="response-types">
 <section id="section-2.4">
         <h3 id="name-response-types">
 <a href="#section-2.4" class="section-number selfRef">2.4. </a><a href="#name-response-types" class="section-name selfRef">Response Types</a>
         </h3>
-<p id="section-2.4-1">A credential request flow MUST use the <a href="https://openid.net/specs/openid-connect-core-1_0.html#CodeFlowSteps">authorization code flow</a> as defined in OpenID Connect core.<a href="#section-2.4-1" class="pilcrow">¶</a></p>
-<p id="section-2.4-2">Given that a credential request flow, results in a credential that MUST be retrieved from the Token Endpoint, the <code>response_type=code</code> parameter MUST be used. Additional <code>response_types</code> in a "hybrid" flow MAY be used; <code>token</code> and <code>id_token</code>; however, this is NOT recommended if these are to contain personally identifiable information about the subject.<a href="#section-2.4-2" class="pilcrow">¶</a></p>
-<p id="section-2.4-3">For mobile applications and SPA's it is RECOMMENDED to follow the use of the [Proof Key Code Exchange (PKCE) by OAuth Clients <code>@!RFC7636</code> protocol.<a href="#section-2.4-3" class="pilcrow">¶</a></p>
+<p id="section-2.4-1">It is RECOMMENDED that a credential request flow use the <a href="https://openid.net/specs/openid-connect-core-1_0.html#CodeFlowSteps">authorization code flow</a> as defined in OpenID Connect core.<a href="#section-2.4-1" class="pilcrow">¶</a></p>
+<p id="section-2.4-2">For instances where <a href="https://openid.net/specs/openid-connect-core-1_0.html#ImplicitFlowAuth">implicit flow</a> is used, the <code>response_type</code> of <code>credential</code> SHOULD be used.<a href="#section-2.4-2" class="pilcrow">¶</a></p>
```

Comment:
credential is a bit of an overloaded term. Could we make the response_type code a bit more specific?

Reply:
Have defined the term credential in terminology, agree its a bit overloaded however think it is the best we can do for now?